docs/usage.rst

```diff
@@ -50,6 +50,16 @@ plugins, so you can just use it:
        fs.create_file('/var/data/xx1.txt')
        assert os.path.exists('/var/data/xx1.txt')
 
+If you don't like the ``fs`` fixture name (``pylint`` may complain about the
+name), you may define your own alias in your ``conftest.py``:
+
+.. code:: python
+
+    @pytest.fixture
+    def fake_filesystem(fs):
```

Comment:
No `# pylint:disable=invalid-name`?

Reply:
"""Variable name 'fs' causes a PyLint warning. Provide a longer name acceptable
to PyLint for use in tests.
"""

Reply:
Thanks! Will make the changes tomorrow.

Reply:
> No # pylint:disable=invalid-name?

We don't use `pylint`, and `flake8` doesn't have problems with this, but we can add it for illustration in this case.

Reply:
Actually you can ignore this comment - I overlooked that this in the documentation, not in the example code. I have added it as proposed.